assets/js/src/form_editor/store/form_body_to_blocks.jsx

```diff
@@ -67,6 +67,22 @@ const mapColorSlug = (colorDefinitions, colorValue) => {
   return result ? result.slug : undefined;
 };
 
+/**
+ * @param {Array.<{name: string, slug: string, size: number}>} fontSizeDefinitions
+ * @param {string} fontSizeValue
+ */
+const mapFontSizeSlug = (fontSizeDefinitions, fontSizeValue) => {
+  let value = 0;
+  if (fontSizeValue) {
+    value = parseInt(fontSizeValue, 10);
+    if (Number.isNaN(value)) {
```

Comment:
Just wondering why do we need to handle this?

Reply:
Just to make sure that we don't break something if a non-numeric value comes from the server.